docs/topics/Gateway.md

```diff
@@ -574,17 +575,17 @@ Sent by the client to indicate a presence or status update.
 
 ###### Gateway Status Update Structure
 
-| Field  | Type                                                     | Description                                                                                 |
-| ------ | -------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
-| since  | ?integer                                                 | unix time (in milliseconds) of when the client went idle, or null if the client is not idle |
-| game   | ?[activity](#DOCS_TOPICS_GATEWAY/activity-object) object | null, or the user's new activity                                                            |
-| status | string                                                   | the user's new [status](#DOCS_TOPICS_GATEWAY/update-status-status-types)                    |
-| afk    | boolean                                                  | whether or not the client is afk                                                            |
+| Field      | Type                                                               | Description                                                                                 |
+|------------|--------------------------------------------------------------------|---------------------------------------------------------------------------------------------|
+| since      | ?integer                                                           | unix time (in milliseconds) of when the client went idle, or null if the client is not idle |
```

Comment:
So what is the implication of sending null for `since` with a status of `"idle"`?